packages/jest-cli/src/cli/index.js

```diff
@@ -297,6 +298,18 @@ const _run = async (
   // Queries to hg/git can take a while, so we need to start the process
   // as soon as possible, so by the time we need the result it's already there.
   const changedFilesPromise = getChangedFilesPromise(globalConfig, configs);
+
+  if (globalConfig.watch && !changedFilesPromise) {
+    console.log(
```

Comment:
We usually do `process.stdout.write`, I think

Reply:
done :)